cf/data/data.py

```diff
@@ -12383,61 +12372,55 @@ def empty(
         units=None,
         calendar=None,
         fill_value=None,
-        chunk=True,
+        chunks=_DEFAULT_CHUNKS,
     ):
-        """Create a new data array without initializing the elements.
+        """Return a new array of given shape and type, without
+        initializing entries.
 
-        Note that the mask of the returned empty data is hard.
-
-        .. seealso:: `full`, `hardmask`, `ones`, `zeros`
+        .. seealso:: `full`, `ones`, `zeros`
 
         :Parameters:
 
             shape: `int` or `tuple` of `int`
                 The shape of the new array.
 
-            dtype: `numpy.dtype` or any object convertible to `numpy.dtype`
-                The data-type of the new array. By default the data-type
-                is ``float``.
+            dtype: data-type
+                The data-type of the new array. By default the
+                data-type is ``float``.
 
             units: `str` or `Units`
                 The units for the new data array.
 
             calendar: `str`, optional
                 The calendar for reference time units.
 
-            fill_value: optional
-                The fill value of the data. By default, or if set to
-                `None`, the `numpy` fill value appropriate to the array's
-                data-type will be used (see
-                `numpy.ma.default_fill_value`). Ignored if the *source*
-                parameter is set.
+            {{chunks: `int`, `tuple`, `dict` or `str`, optional}}
 
-                The fill value may also be set after initialisation with
-                the `set_fill_value` method.
+                .. versionadded:: 4.0.0
 
-                *Parameter example:*
-                  ``fill_value=-999.``
+            fill_value: deprecated at version 4.0.0
+                Use `set_fill_value` instead.
 
         :Returns:
 
             `Data`
+                Array of uninitialized (arbitrary) data of the given
+                shape and dtype, and order.
```

Comment:
```suggestion
                shape and dtype.
```

since I guess reference to 'order' was from copy-and-pasting from `numpy` docstrings and [their `order` keyword](https://numpy.org/doc/stable/reference/generated/numpy.empty.html), which we aren't supporting as far as I see from this PR (though we could maybe eventually add one to these methods to pass through?).